src/components/shared/Loader.vue

```diff
@@ -10,10 +12,11 @@
   border-right: 10px solid #EEEEEE;
   border-bottom: 10px solid #C4D600;
   border-left: 10px solid #EEEEEE;
-  width: 50px;
-  height: 50px;
-  -webkit-animation: spin 1s linear infinite;
-  animation: spin 1s linear infinite;
+  margin: auto;
+  width: 100px;
+  height: 100px;
+  -webkit-animation: spin 2s linear infinite;
+  animation: spin 2s linear infinite;
 }
 
 @-webkit-keyframes spin {
```

Comment:
I'd use just keyframe (without specifying webkit) and better, write all the animations into it's own file the same way you write the colors in it's file